dimensiondata/src/test/java/org/jclouds/dimensiondata/cloudcontrol/features/InfrastructureApiLiveTest.java

```diff
@@ -52,7 +54,8 @@ public void testListOperatingSystems() {
       for (Datacenter dc : getDatacenters()) {
          datacenterIds.add(dc.id());
       }
-      FluentIterable<OperatingSystem> operatingSystems = api().listOperatingSystems(datacenterIds).concat();
+      ImmutableList<OperatingSystem> operatingSystems = api()
+            .listOperatingSystems(DatacenterIdListFilters.Builder.datacenterId(datacenterIds)).toList();
```

Comment:
[minor] static importing the `datacenterId` method makes the invocation cleaner. Not something to be changed, just a comment on how, in general, we use this kind of option objects in jclouds.